### ISSUE TITLE
[TASK] Blueprint Categories Creation

### ISSUE THREAD

**Issue by monaiuu** [tangle-network/dapp#2549]
## Description
To ensure consistency and prevent fragmentation as the Tangle Blueprint Marketplace grows, create a predefined list of categories for classifying Blueprints. This will enable developers to correctly categorize their Blueprints, improve search and filtering functionality, and maintain an organized structure across the marketplace. 
### Related Task:
https://github.com/webb-tools/tangle/issues/763

**Comment by AtelyPham** [comment]
Is this still relevant, @monaiuu?

**Comment by monaiuu** [comment]
yes once we have more Blueprints live, users should be able to browse and filter based on categories.
